### PR TITLE
chore: removed create-play; content available on docs site

### LINKED ISSUE #1099: 🐛 [Bug report]: Documentation - Fixing reference to "How to create a play"
### Descrcibe the bug?

Currently, we refer to the CREATE-PLAY.md file to understand the steps to create a new play.
We have moved this content to our documentation site - https://docs.reactplay.io/

### Steps to reproduce the bug?

-

### Expected behavior

We need to replace any references to this file and point them toward the docs site. Subsequently, we can get rid of the CREATE-PLAY.md altogether.

### Desktop (Please provide your system information)

-

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by supminn**
# Description

The documentation on "How to create a play" has been moved from CREATE-PLAY.md to https://docs.reactplay.io/how-to-guides/how-to-create-play/
This PR contains the changes to remove the CREATE-PLAY.md file and change the references to point to the docs site.

- Fixes #1099 

# Checklist:

- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/common/header/HeaderNav.jsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ const HeaderNav = ({ showBrowse }) => {
               <a
                 className="app-header-btn app-header-btn--primary umami--click--create-play"
                 data-testid="create-btn"
-                href="https://github.com/reactplay/react-play/blob/main/CREATE-PLAY.md"
+                href="https://docs.reactplay.io/How-To-Guides/how-to-create-play"
```
Comment (Sachin-chaurasiya) [inline src/common/header/HeaderNav.jsx:97]: @supminn, I would recommend moving this URL to const and using it everywhere, so if we are changing the URL, we have to make the change in one place only.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: src/common/header/HeaderNav.jsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/common/header/HeaderNav.jsx:97]: > sure, that seems like a better approach.
> 
> One question - On the docs page, I changed the link for `discussions` and added our discord link instead. Should I make this change on these files as well? (I found a few references to discussions such as [https://github.com/reactplay/react-play/discussions/new?category=ideas&title=${idea.title}](https://github.com/reactplay/react-play/discussions/new?category=ideas&title=$%7Bidea.title%7D))

No, I think that is intended to start a new discussion on the repo.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by supminn** [comment]
Hi @atapas 
I have incorporated the changes suggested by @Sachin-chaurasiya 
Could you please provide your review?

**Comment by atapas** [comment]
@supminn there are a few conflicts to take care.

**Comment by supminn** [comment]
> @supminn there are a few conflicts to take care.

Thanks, I have resolved them.

**Review by Sachin-chaurasiya — APPROVED** [review]
Great work @supminn  🙌🏻,
Thanks for the contributions ✨

**Comment by atapas** [comment]
LGTM

**Review by atapas — APPROVED** [review]

**Pull request merged**